specifyweb/frontend/js_src/lib/components/TreeView/Row.tsx

```diff
@@ -53,6 +56,9 @@ export function TreeRow({
   readonly setFocusedRow: (row: Row) => void;
   readonly synonymColor: string;
   readonly treeName: string;
+  readonly isFirst: boolean;
+  readonly isSecondFocused: GetSet<number | undefined>;
```

Comment:
doesn't look right. variables starting with is are of boolean type, not number - rename. `secondTreeFocusedNode`? but really, I don't understand why you need that here